### PR TITLE
Court appearance

### PR DESCRIPTION
**Pull request by retroredge**

### CODE REVIEW

File: src/main/java/uk/gov/justice/digital/delius/data/api/CourtAppearance.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Data
+@Builder
+public class CourtAppearance {
+    private Long courtAppearanceId;
+    private LocalDateTime appearanceDate;
+    private String crownCourtCalendarNumber;
```
Comment (llmikeyj) [inline src/main/java/uk/gov/justice/digital/delius/data/api/CourtAppearance.java]: Number? Or String?

File: src/main/java/uk/gov/justice/digital/delius/data/api/CourtAppearance.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    private String bailConditions;
+    private String courtNotes;
```
Comment (llmikeyj) [inline src/main/java/uk/gov/justice/digital/delius/data/api/CourtAppearance.java]: Note, or notes?

File: src/main/java/uk/gov/justice/digital/delius/data/api/CourtAppearance.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Data
+@Builder
+public class CourtAppearance {
+    private Long courtAppearanceId;
+    private LocalDateTime appearanceDate;
+    private String crownCourtCalendarNumber;
+    private String bailConditions;
```
Comment (llmikeyj) [inline src/main/java/uk/gov/justice/digital/delius/data/api/CourtAppearance.java]: Condition, or Conditions?

File: src/main/java/uk/gov/justice/digital/delius/data/api/CourtAppearance.java [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    private String courtNotes;
+    private Long eventId;
+    private Long teamId;
+    private Long staffId;
+    private Long softDeleted;
```
Comment (llmikeyj) [inline src/main/java/uk/gov/justice/digital/delius/data/api/CourtAppearance.java]: Boolean plz

File: src/main/java/uk/gov/justice/digital/delius/data/api/CourtReport.java [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+@Builder
+public class CourtReport {
+    private Long courtReportId;
+    private LocalDateTime dateRequested;
+    private LocalDateTime dateRequired;
+    private LocalDateTime allocationDate;
+    private LocalDateTime completedDate;
+    private LocalDateTime sentToCourtDate;
+    private LocalDateTime receivedByCourtDate;
+    private String videoLink;
```
Comment (llmikeyj) [inline src/main/java/uk/gov/justice/digital/delius/data/api/CourtReport.java]: Was this possibly a Boolean field?

File: src/main/java/uk/gov/justice/digital/delius/jpa/standard/entity/CourtReport.java [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+public class CourtReport {
+    @Id
+    @Column(name = "COURT_REPORT_ID")
+    private Long courtReportId;
+
+    @Column(name = "COURT_APPEARANCE_ID")
+    private Long courtAppearanceId;
+
+    @ManyToOne
+    @JoinColumn(name = "COURT_APPEARANCE_ID", referencedColumnName="COURT_APPEARANCE_ID", insertable = false, updatable = false)
```
Comment (llmikeyj) [inline src/main/java/uk/gov/justice/digital/delius/jpa/standard/entity/CourtReport.java]: Possibly delete field courtAppearanceId if not needed elsewhere, and remove the insertable & updateable attributes

File: src/main/java/uk/gov/justice/digital/delius/data/api/CourtAppearance.java (hunk 1 above)
Comment (retroredge) [reply src/main/java/uk/gov/justice/digital/delius/data/api/CourtAppearance.java]: OWN_COURT_CALENDAR_NUMBER VARCHAR2(20)

File: src/main/java/uk/gov/justice/digital/delius/data/api/CourtAppearance.java (hunk 2 above)
Comment (retroredge) [reply src/main/java/uk/gov/justice/digital/delius/data/api/CourtAppearance.java]: COURT_NOTES CLOB

File: src/main/java/uk/gov/justice/digital/delius/data/api/CourtAppearance.java (hunk 3 above)
Comment (retroredge) [reply src/main/java/uk/gov/justice/digital/delius/data/api/CourtAppearance.java]: BAIL_CONDITIONS VARCHAR2(256)

File: src/main/java/uk/gov/justice/digital/delius/data/api/CourtReport.java (hunk 5 above)
Comment (retroredge) [reply src/main/java/uk/gov/justice/digital/delius/data/api/CourtReport.java]: VIDEO_LINK VARCHAR2(20)

File: src/main/java/uk/gov/justice/digital/delius/jpa/standard/entity/CourtReport.java (hunk 6 above)
Comment (retroredge) [reply src/main/java/uk/gov/justice/digital/delius/jpa/standard/entity/CourtReport.java]: Yes not needed so deleted

**Pull request merged**